streaming/src/main/scala/org/apache/gearpump/streaming/task/TaskActor.scala

```diff
@@ -254,6 +254,7 @@ class TaskActor(
           case m: Message =>
             count += 1
             onNext(m)
+            processingWatermark = TaskUtil.max(processingWatermark, m.timestamp)
```

Comment:
This doesn't look right for window processing watermark which **can not** advance on every new message